Api/Services/EmploymentService.cs

```diff
@@ -44,4 +52,48 @@ public async Task<Result<bool>> DeleteEmploymentAsync(int employmentId, string u
 
         return true;
     }
+    /// <summary>
+    /// Terminates all employments specified through ids in the list
+    /// </summary>
+    /// <param name="employmentIds"></param>
+    /// <param name="user"></param>
+    /// <returns></returns>
+    public async Task<Result<bool>> DeleteBulkEmploymentAsync(List<int> employmentIds, User user)
+    {
+
+        var employments = new List<Employment>();
+        foreach (var employmentId in employmentIds)
+        {
+            var employment = await context.Employments
+            .Include(e => e.Restaurant)
+            .ThenInclude(r => r.Group)
+            .FirstOrDefaultAsync(e => e.Id == employmentId && e.DateUntil == null);
+
+            if (employment == null)
+            {
+                return new ValidationFailure
+                {
+                    PropertyName = nameof(Employment),
```

Comment:
Employment to nie PropertyName, akurat w tej końcówce jest trudo bo mamy listę zamiast obiektu, PropertyName ma być `"[indexNaLiscie]"`

Reply:
Tam była literówka, to miało być employment, a nie Employment. 
employment jest definiowany kilka linijek wyżej jako konkretny obiekt, a nie lista